yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -43,11 +43,6 @@ class FlowableOperator : public Flowable<D> {
         : flowable_(std::move(flowable)), subscriber_(std::move(subscriber)) {
       assert(flowable_);
       assert(subscriber_);
-
-      // We expect to be heap-allocated; until this subscription finishes (is
-      // canceled; completes; error's out), hold a reference so we are not
-      // deallocated (by the subscriber).
-      Refcounted::incRef(*this);
```

Comment:
Might this mean `delayed destruction` is wanted?
So we delay the destruction up till terminateImpl is being called. Otherwise, if ref count drops to zero, then this object will be deleted immediately. We might be wanting to prevent that to happen.